assets/src/edit-story/app/media/useUploadMedia/useUploadMedia.js

```diff
@@ -40,13 +40,7 @@ import usePreventWindowUnload from '../../../utils/usePreventWindowUnload';
 function useUploadMedia({ media, pagingNum, mediaType, fetchMedia, setMedia }) {
   const { uploadFile } = useUploader();
   const { showSnackbar } = useSnackbar();
-  const {
-    allowedMimeTypes: {
-      image: allowedImageMimeTypes,
-      video: allowedVideoMimeTypes,
-    },
-  } = useConfig();
-  const allowedMimeTypes = [...allowedImageMimeTypes, ...allowedVideoMimeTypes];
+  const { allowedFileTypes = {} } = useConfig();
```

Comment:
```suggestion
  const { allowedFileTypes = [] } = useConfig();
```

Only arrays have a `join` method, not objects. So the default value here should be an empty array.

Reply:
That being said, adding a default value for `allowedFileTypes` might conceal potential bugs. It should always be set in the editor, and for tests it should be mocked.

For example, in `assets/src/edit-story/components/library/test/mediaPane.js`, the `should only display supported mimeTypes and not crash on unsupported` currently fails because `allowedFileTypes` isn't mocked.

Reply:
Good point, removed the default value and mocked it in tests.